### ISSUE TITLE
How to get list items indices under each topic?

### ISSUE THREAD

**Issue by fishfree** [MaartenGr/BERTopic#171]
I followed [this colab tutoria](https://colab.research.google.com/drive/1FieRA9fLdkQEGDIMYl0I3MCjSUKVF8C-?usp=sharing#scrollTo=ScBUgXn06IK6)l.  I want to get list items indices under each topic, but found noway. Thanks in advance!

**Comment by MaartenGr** [comment]
You can use `topics, probs = topic_model.fit_transform(docs)` to extract the topics in `topics`. Each index will match the document in `docs`. In other words, the topic for `docs[0]` can be found in `topics[0]`.

**Comment by fishfree** [comment]
@MaartenGr Thank you! Sorry for my low-skill, I need your  confirmation. I print the topics variable element after "topics, probs = topic_model.fit_transform(docs)" and output as below:
![image](https://user-images.githubusercontent.com/1741341/124716417-a4244300-df36-11eb-8c2b-98efb72e1bcd.png)

Does that mean: docs[1], docs[2], docs[3] and docs[6] are all clustered under topic -1_file_use_any_me?  docs[0] and docs[7] are both clustered under topic 0_jesus_bible_faith_christ?

**Comment by MaartenGr** [comment]
Yes! That is indeed correct. The order of `docs` should match with the order of `topics`. Thus, `topics[0]` belongs to `docs[0]`, `topics[1]` belongs to `docs[1]`, `topics[2]` belongs to `docs[2]`, etc.

**Comment by fishfree** [comment]
@MaartenGr Thank you! After manually read the docs[0] and docs[7], however, they typically should not categorized under  "0_jesus_bible_faith_christ".
![image](https://user-images.githubusercontent.com/1741341/125008572-57a34980-e095-11eb-8d7a-4767810b82f8.png)

Where did i go wrong?

**Comment by MaartenGr** [comment]
Ah, I believe you should re-run `freq = topic_model.get_topic_info(); freq.head(5)` after you have trained the model. The results that you see when you first open the Google Colab are from a previous session. BERTopic is stochastic in nature, which means that everytime you train the model, you will get slightly different results.
